samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/QuestionScoreListener.java

```diff
@@ -228,12 +228,10 @@ public boolean questionScores(String publishedId, QuestionScoresBean bean,
 			}
 			// build a hashMap (publishedItemId, publishedItem)
 			Map<Long, ItemDataIfc> publishedItemHash = pubService.preparePublishedItemHash(publishedAssessment);
-			log.debug("questionScores(): publishedItemHash.size = "
-					+ publishedItemHash.size());
+            log.debug("questionScores(): publishedItemHash.size = {}", publishedItemHash.size());
```

Comment:
Spacing :)

Reply:
one day Jetbrains will sympathize with our desire to keeps tabs and spaces in the same file?!